src/main/java/net/minecraftforge/client/event/ScreenEvent.java

```diff
@@ -1045,10 +1045,23 @@ public void setNewScreen(Screen newScreen)
      */
     public static class Closing extends ScreenEvent
     {
+        @Nullable
+        private final Screen newScreen;
+
         @ApiStatus.Internal
-        public Closing(Screen screen)
+        public Closing(Screen screen, @Nullable Screen newScreen)
         {
             super(screen);
+            this.newScreen = newScreen;
+        }
+
+        /**
+         * @return The screen that will be opened after this screen is closed. May be null.
```

Comment:
Use return braces, also should indicate what `null` means instead of 'may be null'.

Reply:
Would something like `If null then the game will return to the gameplay.` be sufficient? Not sure how else to explain it. Maybe also `if null, no new screen is being opened`.

Reply:
The first imo, the second can be added as an addendum for greater specificity.

Reply:
Updated.